sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -654,7 +670,16 @@ object RapidsConf {
 
       printToggleHeader("Expressions\n")
     }
-    GpuOverrides.expressions.values.toSeq.sortBy(_.tag.toString).foreach(_.confHelp(asTable))
+    val sqlFunctionsByClass = ConfHelper.getSqlFunctionsByClass
+    GpuOverrides.expressions.toSeq.sortBy(_._2.tag.toString).foreach { case (expr, rule) =>
+      val sqlFunctions = sqlFunctionsByClass.get(expr.getCanonicalName)
+      if (sqlFunctions.isDefined) {
+        val nameWithSqlFunctions = s"${rule.confKey}${sqlFunctions.get.mkString(" (", ", ", ")")}"
+        rule.confHelp(asTable, nameWithSqlFunctions)
```

Comment:
I personally don't like the sql functions with the name of the config.  It feels like we are mixing concerns here.  It should either be a part of the description or it should be its own separate field.

Reply:
+1, I think this is interesting enough on its own to be a separate field, `SQL Function` or something similar.

Reply:
Sounds good.